Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_TAG=5.3.0
+ARG BASE_TAG=2019.03
```

Comment:
This will downgrade a lot of packages (all the packages preinstalled in anaconda). This would be quite disruptive for our users if packages are downgraded.

 I would like to better understand the issue since the we have been using this base image for quite a while now.

Feel free to submit the fasttext change alone and assign the anaconda issue to me.

Reply:
`2019.03` is actually newer than `5.3.0` so it is all good.

For reference, anaconda changed their version scheme:
> We are changing versioning in Anaconda Distribution from a major/minor version scheme to a year.month scheme. We made this change to differentiate between the open source Anaconda Distribution and Anaconda Enterprise, our managed data science platform.
-- https://www.anaconda.com/anaconda-distribution-2018-12-released/